### DISCUSSION TITLE
shared preview size is small on receiver phone , where to adjusting it?

### DISCUSSION THREAD

**Discussion by beeium** [immich-app/immich#9877] [Q&A] [answered]
Hi Team:
I am running 1.105.1 version docker, just founded when I share some media to others, the preview size is too small on their mobile phone, can you let me know where can set the preview to full screen on their phone?

of course when click it to display, it's will automaticlly get into normal full screen displayed as well.

![123](https://github.com/immich-app/immich/assets/170703842/6edaefe0-7d29-4288-86aa-529c337d9450)

**Comment by mmomjian** [comment]
In the album view, you get the small previews. In this case the share is just one asset so there is just one preview. Public shares of single items are albums under the hood, basically.

**Reply by beeium** [reply]
Thanks, like you pointed, when I shared more than 1 item, previewing is good.

![1233](https://github.com/immich-app/immich/assets/170703842/7d06e0da-1c92-4278-a246-4ed7409930b2)

**Comment by mmomjian** [comment, marked as answer]
The second example you have shared is in landscape. Those videos are more able to fit the screen.

**Reply by beeium** [reply]
if you like, I suggest adding automatic fitting phone screen like responsive page, no matter just 1 or 2 items shared?

**Reply by beeium** [reply]
@mmomjian  Hi Matthew, I founded new issue, using immich to play video or view photos, it's very bad color looks lost a lot of details compare to original iphone photo app, see compare below.

orginal:


https://github.com/immich-app/immich/assets/170703842/1c6a8739-ca17-472d-95c0-f6f028830888


By immich:


https://github.com/immich-app/immich/assets/170703842/32281b75-3676-4561-8e47-08fdc78d1143
